### DISCUSSION TITLE
Welcome to vitorio-cli Discussions!

### DISCUSSION THREAD

**Discussion by VitorSouza23** [VitorSouza23/vitorio-cli#3] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with other members of our community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ghost** [comment]
vitorio is able to show the values generated by commands and I think that's really useful :smile:

My suggestion is to set the generated value to the clipboard/pasteboard, for example, if I use the following command:

~$ vitorio gen cpf -f
-> 611.818.587-55

Not only vitorio will show the generated value (611.818.587-55), but also it will make me able to use this value when using control + v, so if I use control + v right now, the output will be:  611.818.587-55

How does that sound? I'm not sure if it's a good idea though :sweat_smile:

**Reply by VitorSouza23** [reply]
OMG! Why did not I think about it before? That's amazing idea! I will put it on the next version! (vlw tio Rodrigo!!!)

**Reply by VitorSouza23** [reply]
I've started to search from a clipboard solution, but unfortunately I didn't found any OS agnostic solution (the [Clipboard](https://learn.microsoft.com/pt-br/dotnet/api/system.windows.clipboard?view=windowsdesktop-6.0) only works on Windows and .NET Framework). Do you have any idea to solve this?

Other solutions I've found are libraries that depends shell commands, and it's could bring security issues to the tool.
